gravitee/values.yaml

```diff
@@ -108,16 +116,20 @@ gateway:
     enabled: true
     # Used to create an Ingress record.
     hosts:
-      - api.example.com
+      - apim.example.com
```

Comment:
now I understand why you propose to use subpaths. I would recommend then to make them configurable.

Reply:
Yeah. Mostly, it's to make it easier to expose everything through the Ingress Controller and have a single certificate as well. I will make it configurable.